protocol/app/ante/sigverify_test.go

```diff
@@ -25,160 +19,6 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-func TestSetPubKey(t *testing.T) {
```

Comment:
Noob question: why is it okay to drop these tests?

Reply:
We are verifying functionality of stuff imported from `sdkante` which is from the Cosmos SDK and not verifying stuff that we modified since this test file was a duplicate of the Cosmos SDK test file for this space. The only thing that was modified was the imports.

Note that we don't have a forked version of the `SetPubKey` ante handler.